contrib/golang/filters/http/test/test_data/passthrough/go.mod

```diff
@@ -2,6 +2,8 @@ module example.com/passthrough
 
 go 1.18
 
-require github.com/envoyproxy/envoy v1.24.0
+require github.com/envoyproxy/envoy/contrib/golang v1.24.0
```

Comment:
I prefer to keep the new changes since the old version doesn't work for now, we'd better make it work for now. And change it when the extension moves to standard.

Since the test cases may also be examples for users, making it work for now could be a better choice.
Users also have to change it when the extension moves to standard.